src/Makemodule-local.am

```diff
@@ -319,30 +319,59 @@ install-exec-hook-cihelpers:
 
 install-exec-hook: install-exec-hook-tntnet-so install-exec-hook-bioscsv install-exec-hook-cihelpers
 
+### This file used to contain our default TNTNET settings, but was
+### later separated into better manageable components. Just in case
+### we have some users for it (such as the mangled test-run file
+### below, we have a recipe to reconstruct it back from pieces in
+### specified order).
+TNTNET_PIECES = \
+	00_start.xml \
+	10_common_statics.xml \
+	20_common_basics.xml \
+	50_main_api_start.xml \
+	51_asset_api_fty-rest.xml \
+	89_main_api_end.xml \
+	99_end.xml
+
+src/web/tntnet.xml: $(addprefix src/web/,$(TNTNET_PIECES))
+	cat $^ > $@
+
 ### This file contains the TNTNET settings relevant for our test runs
 ### Note: in non-daemon mode, stderr is logged to stderr
 tntnet.xml: src/web/tntnet.xml Makefile
-	${SED} -e 's|\(.*\)\(<!--.*<dir>/</dir>.*-->.*\)|\1<dir>$(abs_top_srcdir)/src/web</dir>\n\1<compPath><entry>$(abs_top_builddir)/src/.libs</entry><entry>$(abs_top_builddir)/.libs</entry></compPath>|' $< > $@ || \
-	rm -f $@
+	${SED} -e 's|\(.*\)\(<!--.*<dir>/</dir>.*-->.*\)|\1<dir>$(abs_top_srcdir)/src/web</dir>\n\1<compPath><entry>$(abs_top_builddir)/src/.libs</entry><entry>$(abs_top_builddir)/.libs</entry></compPath>|' $< > $@ \
+	|| rm -f $@
 
 ### This file contains the TNTNET settings distributed as an example for users
 ### Note: in daemon mode, "stderr > /dev/null" unless errorLog is defined
 tntnet.xml.example: src/web/tntnet.xml Makefile
+	$(call sed-tntnet-example,$<,$@)
+
+99_end.xml.example: src/web/99_end.xml Makefile
+	$(call sed-tntnet-example,$<,$@)
+
+### Generate $2 from $1 in a way expected by OS image bundling
+define sed-tntnet-example
 	${SED} -e 's|\(.*\)\(<!--.*<dir>/</dir>.*-->.*\)|\1<dir>$(datarootdir)/@PACKAGE@/web</dir>\n\1<compPath>\n\1\ \ \ <entry>$(bios_libdir)</entry>\n\1\ \ \ <entry>$(pkglibdir)</entry>\n\1\ \ \ <entry>$(libdir)</entry>\n\1\ \ \ <entry>$(bios_libexecdir)</entry>\n\1</compPath>|' \
 	       -e 's|<!-- <errorLog>/var/log/tntnet/error.log</errorLog> -->|<errorLog>/var/log/tntnet/error.log</errorLog>|' \
 	       -e 's|<port>8000</port>|<port>80</port>|' \
 	       -e 's|<port>8443</port>|<port>443</port>|' \
 	       -e 's|<!-- ssl start.*|<!-- ssl start -->|' \
 	       -e 's|^\([[:blank:]]*\)ssl end -->|\1<!-- ssl end -->|' \
-	       -e 's|<!--\ <daemon>0</daemon>\ -->|<daemon>1</daemon>|' $< > $@ || \
-	rm -f $@
+	       -e 's|<!--\ <daemon>0</daemon>\ -->|<daemon>1</daemon>|' $(1) > $(2) \
+	|| rm -f $(2)
+endef
+
+### Most of the example files come unmodified
+%.xml.example: src/web/%.xml
+	cat < $< > $@
```

Comment:
Why not `cp` here ?

Reply:
No apparent reason, copy-pasta from other more complex XML concatenations... Does it impact much?